src/components/Footer.astro

```diff
@@ -0,0 +1,52 @@
+---
+const {} = Astro.props;
+---
+
+<div class="footer-wrapper">
+  <!-- footer wrapper -->
+  <hr />
+  <div class="container">
+    <footer>
+      <ul class="list-inline pull-right">
+        <li style="position: relative; top: 5px">
+          <a href="https://seqera.io" target="_blank" title="Developed by Seqera">
+            <img src="/img/seqera-logo-wide_new.png" alt="Seqera" width="136" height="28" />
+          </a>
+        </li>
+      </ul>
+      <ul class="list-inline pull-left">
+        <li style="position: relative; top: 5px">
+          <a href="https://twitter.com/nextflowio" class="twitter-follow-button" data-show-count="false">
+            Follow @nextflowio
+          </a>
+        </li>
+
+        <li>
+          <a href="mailto:info@nextflow.io">info@nextflow.io</a>
+        </li>
+      </ul>
+      <div class="clearfix"></div>
+    </footer>
+  </div>
+</div>
+<!-- / footer wrapper -->
+
+<!-- Bootstrap core JavaScript
+================================================== -->
+<!-- Placed at the end of the document so the pages load faster -->
+<script is:inline src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

Comment:
are you sure you want to load jquery on every page?

Reply:
The `custom.js` file has a bunch of jQuery stuff in it so yes, for now. There is plenty of front-end refactoring to be done on the site if we want, but I didn't tackle any of that in this PR. Just the site build.

Reply:
okay, I see that's needed for custom.js (which should be component level scripts imo)